### ISSUE TITLE
Blogpost link in last slide for CIO Council

### ISSUE THREAD

**Issue by wslack** [18F/slides#4]
Shouldn't that link say "The Agile BPA is here!" @NoahKunin?
https://18f.gsa.gov/2015/06/15/agile-bpa-is-here/

**Comment by NoahKunin** [comment]
Yep, was waiting on the actual blog post title to change via another PR I
filed last night. Which is now done! In my next edit, good catch, closing.

On Wed, Jul 22, 2015 at 10:52 AM, Will Slack notifications@github.com
wrote:

> Shouldn't that link say "The Agile BPA is here!" @NoahKunin
> https://github.com/NoahKunin?
> https://18f.gsa.gov/2015/06/15/agile-bpa-is-here/
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/18F/slides/issues/4.

## 

Noah Kunin
Delivery Architecture + Infrastructure Services Director
202-577-7167 | @18F https://twitter.com/18F
